secp256k1-sys/src/lib.rs

```diff
@@ -454,6 +454,36 @@ impl KeyPair {
             pk
         }
     }
+
+    /// This function attempts to erase the contents of the underlying array.
+    /// Note, however, that since this type is `Copy`, ensuring that the contents
+    /// of this array don't end up elsewhere in memory is very subtle. For more
+    /// discussion on this, please see the documentation of the
+    /// [`zeroize`](https://docs.rs/zeroize) crate.
```

Comment:
```suggestion
    /// Attempts to erase the contents of the underlying array.
    ///
    /// Note, however, that since this type is `Copy`, ensuring that the contents
    /// of this array don't end up elsewhere in memory is very subtle. For more
    /// discussion on this, please see the documentation of the
    /// [`zeroize`](https://docs.rs/zeroize) crate.
```

Reply:
:+1: incorporated in latest squash (along with next two suggestions) --- thank you!